pkg/auth/auth.go

```diff
@@ -166,13 +166,17 @@ func RenewService(client *api.Client, config cfg.Config) {
 			log.Fatalln("The existing token failed renewal, exiting..")
 		}
 		ttl, err := result.TokenTTL()
+		if err != nil {
+			log.Fatalln("Failed to parse the token's ttl from JSON")
+		}
```

Comment:
I'm happy to improve the error the message if you have suggestions.

Reply:
Works for me!